kong/plugins/file-log/handler.lua

```diff
@@ -69,6 +73,12 @@ local FileLogHandler = {
 
 
 function FileLogHandler:log(conf)
+  if conf.custom_fields_by_lua then
+    for key, expression in pairs(conf.custom_fields_by_lua) do
+      kong.log.set_serialize_value(key, sandbox.sandbox(expression, sandbox_opts)())
```

Comment:
`kong.log.set_serialize_value` can be cached inside the `:log` handler to speed up the hot path (it can not be done in the module level because the namespaces log object may not been created yet at the time of loading this file).

`sandbox.sandbox` can be cached at module top level to speed up the hot path.